Missing ament_cmake_gtest package

See this [PR](https://github.com/ros-tooling/system_metrics_collector/pull/5/checks?check_run_id=289704468) for details

```
CMake Error at CMakeLists.txt:38 (find_package):
  By not providing "Findament_cmake_gtest.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "ament_cmake_gtest", but CMake did not find one.

  Could not find a package configuration file provided by "ament_cmake_gtest"
  with any of the following names:

    ament_cmake_gtestConfig.cmake
    ament_cmake_gtest-config.cmake

  Add the installation prefix of "ament_cmake_gtest" to CMAKE_PREFIX_PATH or
  set "ament_cmake_gtest_DIR" to a directory containing one of the above
  files.  If "ament_cmake_gtest" provides a separate development package or
  SDK, be sure it has been installed.
```

Also happening in launch_ros_sandbox#69

```
2019-11-06T03:21:25.1682260Z Finished <<< ament_cmake_core [3.31s]
2019-11-06T03:21:25.1688740Z Starting >>> ament_cmake_libraries
2019-11-06T03:21:25.3650500Z --- output: ament_cmake_libraries
2019-11-06T03:21:25.3650740Z CMake Error at CMakeLists.txt:5 (find_package):
2019-11-06T03:21:25.3650850Z   By not providing "Findament_cmake_core.cmake" in CMAKE_MODULE_PATH this
2019-11-06T03:21:25.3650960Z   project has asked CMake to find a package configuration file provided by
2019-11-06T03:21:25.3651060Z   "ament_cmake_core", but CMake did not find one.
2019-11-06T03:21:25.3651110Z 
2019-11-06T03:21:25.3651210Z   Could not find a package configuration file provided by "ament_cmake_core"
2019-11-06T03:21:25.3651300Z   with any of the following names:
2019-11-06T03:21:25.3651350Z 
2019-11-06T03:21:25.3651470Z     ament_cmake_coreConfig.cmake
2019-11-06T03:21:25.3651920Z     ament_cmake_core-config.cmake
2019-11-06T03:21:25.3651970Z 
2019-11-06T03:21:25.3652070Z   Add the installation prefix of "ament_cmake_core" to CMAKE_PREFIX_PATH or
2019-11-06T03:21:25.3652170Z   set "ament_cmake_core_DIR" to a directory containing one of the above
2019-11-06T03:21:25.3652270Z   files.  If "ament_cmake_core" provides a separate development package or
2019-11-06T03:21:25.3652370Z   SDK, be sure it has been installed.
2019-11-06T03:21:25.3652410Z 
2019-11-06T03:21:25.3652460Z 
2019-11-06T03:21:25.3652860Z -- Configuring incomplete, errors occurred!
2019-11-06T03:21:25.3652980Z See also "/Users/runner/runners/2.160.0/work/launch_ros_sandbox/launch_ros_sandbox/ros2_ws/build/ament_cmake_libraries/CMakeFiles/CMakeOutput.log".
2019-11-06T03:21:25.3653360Z ---
2019-11-06T03:21:25.3654070Z --- stderr: ament_cmake_libraries
2019-11-06T03:21:25.3654180Z CMake Error at CMakeLists.txt:5 (find_package):
2019-11-06T03:21:25.3654290Z   By not providing "Findament_cmake_core.cmake" in CMAKE_MODULE_PATH this
2019-11-06T03:21:25.3654390Z   project has asked CMake to find a package configuration file provided by
2019-11-06T03:21:25.3654480Z   "ament_cmake_core", but CMake did not find one.
2019-11-06T03:21:25.3654540Z 
2019-11-06T03:21:25.3654630Z   Could not find a package configuration file provided by "ament_cmake_core"
2019-11-06T03:21:25.3654710Z   with any of the following names:
2019-11-06T03:21:25.3654770Z 
2019-11-06T03:21:25.3654850Z     ament_cmake_coreConfig.cmake
2019-11-06T03:21:25.3655240Z     ament_cmake_core-config.cmake
2019-11-06T03:21:25.3655310Z 
2019-11-06T03:21:25.3655400Z   Add the installation prefix of "ament_cmake_core" to CMAKE_PREFIX_PATH or
2019-11-06T03:21:25.3655500Z   set "ament_cmake_core_DIR" to a directory containing one of the above
2019-11-06T03:21:25.3655590Z   files.  If "ament_cmake_core" provides a separate development package or
2019-11-06T03:21:25.3655680Z   SDK, be sure it has been installed.
2019-11-06T03:21:25.3655990Z 
2019-11-06T03:21:25.3656030Z 
2019-11-06T03:21:25.3656430Z ---
```

--- a/lib/action-ros2-ci.js
+++ b/lib/action-ros2-ci.js
@@ -74,6 +74,20 @@ EOF`], options);
             if (colconMixinName !== "") {
                 extra_options.concat(["--mixin", colconMixinName]);
             }
+            // Add the future install bin directory to PATH.
+            // This enables cmake find_package to find packages installed in the
+            // colcon install directory, even if local_setup.sh has not been sourced.
+            //
+            // From the find_package doc:
+            // https://cmake.org/cmake/help/latest/command/find_package.html
+            //   5. Search the standard system environment variables.
+            //   Path entries ending in /bin or /sbin are automatically converted to
+            //   their parent directories:
+            //   PATH
+            //
+            // ament_cmake should handle this automatically, but we are seeing cases
+            // where this does not happen. See issue #26 for relevant CI logs.
+            core.addPath(path.join(ros2WorkspaceDir, "install", "bin"));
             yield exec.exec("colcon", ["build", "--event-handlers", "console_cohesion+", "--packages-up-to",
                 packageName, "--symlink-install"].concat(extra_options), options);
             yield exec.exec("colcon", ["test", "--event-handlers", "console_cohesion+", "--pytest-args",

--- a/src/action-ros2-ci.ts
+++ b/src/action-ros2-ci.ts
@@ -79,6 +79,22 @@ EOF`], options);
     if (colconMixinName !== "") {
       extra_options.concat(["--mixin", colconMixinName]);
     }
+
+    // Add the future install bin directory to PATH.
+    // This enables cmake find_package to find packages installed in the
+    // colcon install directory, even if local_setup.sh has not been sourced.
+    //
+    // From the find_package doc:
+    // https://cmake.org/cmake/help/latest/command/find_package.html
+    //   5. Search the standard system environment variables.
+    //   Path entries ending in /bin or /sbin are automatically converted to
+    //   their parent directories:
+    //   PATH
+    //
+    // ament_cmake should handle this automatically, but we are seeing cases
+    // where this does not happen. See issue #26 for relevant CI logs.
+    core.addPath(path.join(ros2WorkspaceDir, "install", "bin"));
+
     await exec.exec(
         "colcon",
         ["build", "--event-handlers", "console_cohesion+", "--packages-up-to",

